Homework_21/script.js

```diff
@@ -0,0 +1,12 @@
+$(function () 
```

Comment:
I like that you know already jQuery library, but you didn't need it for just a calculate some values